engine/src/flutter/shell/platform/embedder/embedder.h

```diff
@@ -105,6 +105,8 @@ typedef enum {
   kFlutterAccessibilityFeatureHighContrast = 1 << 5,
   /// Request to show on/off labels inside switches.
   kFlutterAccessibilityFeatureOnOffSwitchLabels = 1 << 6,
+  /// Indicate the platform does not support announcements.
+  kFlutterAccessibilityFeatureNoAnnounce = 1 << 7,
```

Comment:
embedder API codeowners review: this is safe and will not break ABI compatibility.